admin_manual/configuration/server/config_sample_php_parameters.rst

```diff
@@ -722,32 +722,38 @@ The default value is ``ownCloud``.
 			'shared_secret' => '57b58edb6637fe3059b3595cf9c41b9',
 			'users' => ['user1'],
 			'apps' => ['files_texteditor'],
-			'logfile' => '/tmp/test.log'
+			'logfile' => '/tmp/test1.log'
 	        ],
 	        [
 			'shared_secret' => '57b58edb6637fe3059b3595cf9c41b9',
-			'users' => ['user1'],
+			'users' => ['user2'],
 			'apps' => ['gallery'],
-			'logfile' => '/tmp/gallery.log'
+			'logfile' => '/tmp/test2.log'
 	        ],
 	],
 
-Log condition for log level increase based on conditions. Once one of these
-conditions is met, the required log level is set to debug. This allows to
-debug specific requests, users or apps
+Log condition for log level increase based on conditions. 
+You can populate one, two or all three conditions.
+The first met condition inside a condition block sets the log level to debug and writes a log entry if avaliable.
+This allows to debug specific requests, users or apps.
```

Comment:
s/This allows to debug specific/This allows for debugging of specific/g

Reply:
What about:
This allows to debug specific requests, users or apps ->
This allows debugging specific requests, users or apps
?